lazyflow/operators/opSlicedBlockedArrayCache.py

```diff
@@ -138,8 +138,8 @@ def setupOutputs(self):
         ram_per_pixel = 0
         if self.Output.meta.dtype == object or self.Output.meta.dtype == numpy.object_:
             ram_per_pixel = sys.getsizeof(None)
-        elif numpy.issubdtype(self.Output.meta.dtype, numpy.dtype):
-            ram_per_pixel = self.Output.meta.dtype().nbytes
+        elif numpy.issubdtype(self.Output.meta.dtype, numpy.generic):
+            ram_per_pixel = numpy.dtype(self.Input.meta.dtype).itemsize
```

Comment:
I see this `ram_per_pixel` value is now based on the size of `self.Input.dtype` rather than `self.Output.dtype`. Is it correct to have `ram_per_pixel` be base on `Input` some times and on `Output` other times?

Reply:
that was a type, good catch!

Reply:
on the other hand, the cache doesn't change data-type. So technically it doesn't make a difference. The whole surrounding code also only checks the input. So for now I'd rather leave it like this than going for either a) inconsistency with the surrounding code, or b) polluting this PR with unrelated changes. Are you ok with that?

Reply:
Sure. Let's keep it simple and reviewable. We'll get back to it later =)